types/index.d.ts

```diff
@@ -3024,6 +3024,7 @@ export declare namespace Knex {
     connectionTimeoutMillis?: number;
     types?: PgCustomTypesConfig;
     options?: string;
+    expirationChecker?(): boolean;
   }
```

Comment:
can you add tsd test for this?

Reply:
I'll try. If i'm honest, i didn't do it because I didn't see those tests for the types that already define `expirationChecker`, so this maintains the current standard. I do appreciate it's hard to improve things if we only maintain the current standards tho.

Reply:
@kibertoad i've not done TSD tests before - is this right? https://github.com/knex/knex/pull/5334/commits/2e7fca0093ee1662ede628c7c5dbb72e31f9a7f6